src/elm/Api/Deserialize.elm

```diff
@@ -33,17 +33,14 @@ type alias Admin =
 type alias Site =
     { id : String
     , hostname : String
-    , verified : Bool
 
     -- UUID value
     -- choosing not to type as UUID using something like
     -- NoRedInk/elm-uui because we will not be manipulating
     -- this value.
     -- It will only be rendered to the user
```

Comment:
These comments can also be removed snice they were context on the `dnsTag` field.